docs/docs/guides/sso.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: Configuring Single sign-on
+---
+# Configuring Single sign-on
+
+In Infrahub you can configure SSO using either Open ID Connect (OIDC), alternatively you can use OAuth2.
+
+The SSO system on with Infrahub allows for the configuration of one or more identity providers. While most organizations will only use one provider a reason to have two could be that the providers manage different security domains where one of them might be for regular users the other identity provider could be for administrative accounts.
```

Comment:
The SSO system ~on with~ in Infrahub allows for the configuration of one or more identity providers. While most organizations will only use one provider a reason to have two could be that the providers manage different security domains where one of them might be for regular users the other identity provider could be for administrative accounts.